### DISCUSSION TITLE
Welcome to lunatic Discussions!

### DISCUSSION THREAD

**Discussion by bkolobara** [lunatic-solutions/lunatic#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you are doing with Lunatic.

**Comment by zicklag** [comment]
:wave: Hey there, I just found Lunatic a bit ago and haven't used it or even downloaded it, but I thought it was a very cool and interesting idea! I might try it out if I ever get the time to try it for a DevOps orchestration tool that me and my team wanted to build on top of an actor system. You never know. :wink:

Anyway, neat project!

**Comment by betamos** [comment]
Hi 👋 . I work on Fuchsia which has a large, mostly async, codebase.

I am interested in this project primarily due to the concurrency model. There are a few ergonomics and compatibility issues with async that I think should ideally not be user facing. In terms of ergonomics, async pushes advanced language features directly onto the user, such as lifetimes and pinning. In terms of compatibility, async and sync functions are fundamentally part of the API surface, which creates ecosystem fragmentation. The coroutine model, where continuation is an implementation detail, eliminates these problems rather efficiently. Additionally, message passing for synchronization tends to be easier to use and less error-prone than shared mutable state (such as mutices), which Go, Erlang and other CSP-like models demonstrate. So in short, great design choices!

I'm not planning on using Lunatic for anything at the moment, so really I'm just lurking.
